packages/grafana-prometheus/src/querybuilder/components/PromQueryCodeEditor.tsx

```diff
@@ -35,7 +34,6 @@ export function PromQueryCodeEditor(props: PromQueryCodeEditorProps) {
         app={app}
       />
       {showExplain && <PromQueryBuilderExplained query={query.expr} />}
-      <QueryEditorHints query={query} datasource={datasource} data={data} onChange={onChange} onRunQuery={onRunQuery} />
```

Comment:
This was unnecessary since rendering the hint in code mode is handled in `PromQueryField` component. This additional component renders the same hint again which is creating confusion. So I removed the component.